packages/cli/test/cli.test.js

```diff
@@ -93,22 +93,18 @@ describe("CLI", () => {
     timeout
   );
 
-  /**
-   * Skip test until circleci is reconfigured to run in macos and windows envs
-   */
-  xtest(
-    "should successfully build a mobi",
-    async done => {
-      const testFilePath = path.join("static", "downloads", "test");
-      await quire.mobi(testFilePath);
-      assert.equal(
-        fs.existsSync(`${testFilePath}.mobi`),
-        true
-      );
-      done();
-    },
-    timeout
-  );
+  if (process.platform !== "linux") {
```

Comment:
A comment here about why we are skipping this test would be kind to our future selves.